pkg/bootstrap/bootstrap.go

```diff
@@ -35,24 +38,45 @@ func Write(w io.Writer, bootstrap *config.ControlRuntimeBootstrap) error {
 			return errors.Wrapf(err, "failed to read %s", path)
 		}
 
-		dataMap[pathKey] = data
+		info, err := os.Stat(path)
+		if err != nil {
+			return err
+		}
+
+		dataMap[pathKey] = File{
+			Timestamp: info.ModTime(),
+			Content:   data,
+		}
 	}
 
 	return json.NewEncoder(w).Encode(dataMap)
 }
 
-func Read(r io.Reader, bootstrap *config.ControlRuntimeBootstrap) error {
-	paths, err := objToMap(bootstrap)
+// File is a representation of a certificate
+// or key file within the bootstrap context that contains
+// the contents of the file as well as a timestamp from
+// when the file was last modified.
+type File struct {
+	Timestamp time.Time
+	Content   []byte
+}
+
+type DataFormat map[string]File
```

Comment:
This feels weird to me and doesn't say much about what it contains, what about something like
```suggestion
type FilePathMap map[string]File
```

Reply:
Go prefers not to use Hungarian notation as the type is known throughout.

Reply:
Fair enough, but the current name seems awfully generic for something that we're bothering to create a type for.

Reply:
updated